TeaDiary/Scripts/App/Teas/Details.js

```diff
@@ -64,13 +64,13 @@ function onUpdateClick() {
     }
 
     apiCommands.updateTea(tea,
-        (success) => {
-            if (success) {
-                showSuccessMessage();
-            } else {
-                showFailureMessage();
-            }
-        });
+        success => {
+            showSuccessMessage();
+        },
```

Comment:
I think `success => showSuccessMessage()` will work. This is an exception to a rule to always write {}